packages/labs/analyzer/test-files/ts/functions/src/functions.ts

```diff
@@ -36,3 +36,18 @@ export function function2(a: string, b = false, ...c: number[]) {
 export default function () {
   return 'default';
 }
+
+export function overloaded(x: string): string;
+export function overloaded(x: number): number;
+/**
+ * This function has an overloaded signature in TS.
+ * @param x Some value, either a string or a number.
+ * @returns Returns either a string or a number.
+ */
```

Comment:
There seems to be a bug where the descriptions (etc.) aren't picked up if this comment isn't attached to the main declaration.

Reply:
This was removed.